### PR TITLE
MM-47938 Ensure feature flag is set before running E2E test

### PR DESCRIPTION
**Pull request by hmhealey**
This test is still failing on nightly builds, and I think it might be because the feature flag is missing for this. Since the feature flag is now on-by-default on master, that should be fixed for us, but I wanted to be extra sure so I changed the test to fail if it's not enabled.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-47938

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/channel/new_channel_with_board_spec.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
```
Comment (jgilliam17) [inline e2e/cypress/tests/integration/channel/new_channel_with_board_spec.js:9]:
```suggestion

// Stage: @prod
```

**Comment by jgilliam17** [comment]
Thanks @hmhealey 
Test passes see full report here https://mattermost-cypress-report.s3.amazonaws.com/287416-2915f68-onprem-ent-webapp-pr-11965-mm-ee-test:2915f68/mochawesome.html
Screen Shot 2023-01-11 at 11 39 16 AM

**Review by jgilliam17 — APPROVED** [review]

**Comment by jgilliam17** [comment]
@amyblais do we need to change fix version on this ticket, should I set for next cloud ?

**Comment by amyblais** [comment]
@jgilliam17 Sure, I've fixed the fix version now.

**Comment by jgilliam17** [comment]
Thanks Amy!

**Pull request merged**